cfme/tests/configure/test_tag.py

```diff
@@ -50,6 +52,30 @@ def test_tag_crud(category):
     tag.delete()
 
 
+def test_map_tagging_crud(appliance, category, soft_assert):
+    """Test Mam tagging grud with flash message assertion"""
+    label = fauxfactory.gen_alphanumeric(8)
+    random_category_name = fauxfactory.gen_alphanumeric(8)
+    map_tags_collection = appliance.collections.map_tags
+    map_tag_entity = map_tags_collection.create('Container Project', label, category.name)
+
+    view = appliance.browser.create_view(navigator.get_class(map_tags_collection, 'All').VIEW)
+    view.flash.assert_success_message('Container Label Tag Mapping "{}" was added'.format(label))
+
+    map_tag_entity.update({'category': random_category_name})
+
+    view = appliance.browser.create_view(navigator.get_class(map_tags_collection, 'All').VIEW)
+    view.flash.assert_success_message('Container Label Tag Mapping "{}" was saved'.format(label))
+    row = view.table.rows(resource_label=label)
+    soft_assert(row.tag_category.text == 'random_category_name')
+
+    map_tag_entity.delete()
+    view = appliance.browser.create_view(navigator.get_class(map_tags_collection, 'All').VIEW)
+    if not BZ(1510473, forced_streams=['5.9']).blocks:
```

Comment:
This BZ is closed as a duplicate of 1513507, that one is also closed, `if` is not needed here